k8s/install/etc/config.sh.template

```diff
@@ -11,13 +11,18 @@ export KOPS_ZONES="us-east-1b,us-east-1c,us-east-1d"
 export KOPS_MASTER_ZONES="us-east-1b,us-east-1c,us-east-1d"
 export TF_STATE_BUCKET="${KOPS_SHORT_NAME}-tf-state"
 
+# populate these if installing FluentD->PaperTrail DaemonSet
+export PAPERTRAIL_HOST=""
+export PAPERTRAIL_PORT=""
```

Comment:
NIT: these could work with any remote syslog, perhaps we should rename these to `SYSLOG_HOST` and `SYSLOG_PORT`?